### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by heimrih**
ready

### CODE REVIEW

**Comment by heimrih** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
```
Comment (quoctd-0607) [inline Gemfile]: using double quote. 
' ' => " "
fix all in this file

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```
Comment (ngocvt-0484) [inline Gemfile]: use gem mysql2 instead of sqlite3

File: config/database.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```
Comment (quoctd-0607) [inline config/database.yml]: this file not push to git. you can create database.yml.example and push it

File: Gemfile [hunk 4]
```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```
Comment (ngocvt-0484) [inline Gemfile]: remove all comments

File: Gemfile [hunk 5]
```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.1'
```
Comment (ngocvt-0484) [inline Gemfile]: in rails, we will use " instead of '
please change all files

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (ngocvt-0484) [inline app/views/layouts/application.html.erb]: use "

**Review by quoctd-0607 — COMMENTED** [review]
test file, tmp file, public/system, public/assets you can define in gitignor

File: config/database.yml [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
```
Comment (ngocvt-0484) [inline config/database.yml]: this file will not be up to GIt, please 
cp database.yml database.yml.example
git rm --cached config/database.yml

File: config/locales/en.yml [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+# To use a different locale, set it with `I18n.locale`:
+#
+#     I18n.locale = :es
+#
+# This would use the information in config/locales/es.yml.
+#
+# The following keys must be escaped otherwise they will not be retrieved by
+# the default I18n backend:
+#
+# true, false, on, off, yes, no
```
Comment (ngocvt-0484) [inline config/locales/en.yml]: remove all comments

File: config/database.yml (hunk 7 above)
Comment (ngocvt-0484) [reply config/database.yml]: and in .gitignore add
/config/database.yml

**Pull request merged**